### PR TITLE
Dockerfile.gcc: Change rtw88 revision

### PR DESCRIPTION
**Pull request by rene**
Update rtw88 to a tagged revision which is known to build successfully with our kernel.

~~The current rtw88 WiFi driver repository doesn't have any tags or branches fixed to the revision used by EVE and the current master branch is broken (doesn't build with eve-kernel). This commit moves the rtw88 to an alternative repository, with the right revision tagged.~~

### CODE REVIEW

**Review by uncleDecart — APPROVED** [review]
LGTM, let's see if PR works, and also do we want to move this fork under lf-edge?

**Comment by rene** [comment]
> LGTM, let's see if PR works, and also do we want to move this fork under lf-edge?

No, I will work on a fix (to make current rtw88 master branch work) and ask maintainers create a tag that we can use. Once everything is fixed upstream, we can switch back to the original repo.

**Comment by rene** [comment]
I've opened an upstream PR fixing the build issue and asking for tagging: https://github.com/lwfinger/rtw88/pull/191
Let's wait for review, once it gets merged, we can keep the upstream repository...

**Comment by rene** [comment]
My upstream PR was merged and now we have a tagged version to point it out. This PR was updated and is ready for review.

**Comment by rucoder** [comment]
@rene are you sure this PR is correct? 10000+ commits

**Comment by rene** [comment]
> @rene are you sure this PR is correct? 10000+ commits

@rucoder , I have no idea what was going on.... will fix that, it should have only a single small commit in this PR....

**Comment by rene** [comment]
@rucoder , it's fixed.

**Review by rucoder — APPROVED** [review]
LGTM!

**Pull request merged**
